src/riak_repl_aae_sink.erl

```
@@ -108,8 +108,8 @@ handle_info({Error, Socket, Reason},
     lager:info("AAE sink connection to ~p closed unexpectedly: ~p",
                [State#state.clustername, {Socket, Error, Reason}]),
     {stop, {Socket, Error, Reason}, State};
-handle_info(_Info, State) ->
-    {noreply, State}.
+handle_info({hashtree_lock,_}, State) ->
+    send_reply(ok, State).
```

Same thing with the catch all clause here. Let's add it back.